RGB.NET.Layout/DeviceLayout.cs

```diff
@@ -14,7 +14,23 @@ namespace RGB.NET.Layout;
 /// </summary>
 [Serializable]
 [XmlRoot("Device")]
-public class DeviceLayout : IDeviceLayout
+public class DeviceLayout : DeviceLayout<object, object>
```

Comment:
I'm not a 100% sure on this, it's a nice shortcut to deal with layouts if you're not interested in custom data but it creates this scenario:

```cs
DeviceLayout layoutA = new DeviceLayout();
DeviceLayout<object, object> layoutB = new DeviceLayout<object, object>();

layoutA = layoutB; // not possible as you cannot assign DeviceLayout<object, object> to DeviceLayout
layoutB = layoutA; // possible because DeviceLayout is a derived type of DeviceLayout<object, object>
```